docs/serving/samples/cloudevents/cloudevents-nodejs/index.js

```diff
@@ -15,11 +15,10 @@ limitations under the License.
 */
 
 const express = require('express')
-const { CloudEvent, HTTPEmitter, HTTPReceiver } = require('cloudevents-sdk')
+const { CloudEvent, Emitter, HTTP, headersFor } = require('cloudevents')
```

Comment:
Small nit `headersFor` is an unused variable

Reply:
Fixed